scripts/test/run-tests.sh

```diff
@@ -38,9 +38,9 @@ BBB_DEBIAN_EMMC_IMAGE_URL="https://rcn-ee.com/rootfs/bb.org/testing/2022-03-02/b
 ## Auto-update
 RASPBIAN_IMAGE_URL="https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2022-09-26/2022-09-22-raspios-bullseye-armhf-lite.img.xz"
 
-UBUNTU_IMAGE_URL="https://downloads.mender.io/mender-convert/images/Ubuntu-Jammy-x86-64.img.gz"
+UBUNTU_IMAGE_URL="https://downloads.mender.io/mender-convert/images/wip-MEN-6282-Ubuntu-Jammy-x86-64.img.gz"
 
-DEBIAN_IMAGE_URL="https://downloads.mender.io/mender-convert/images/Debian-11-x86-64.img.gz"
+DEBIAN_IMAGE_URL="https://downloads.mender.io/mender-convert/images/wip-MEN-6282-Debian-11-x86-64.img.gz"
```

Comment:
Note to self. After review,
* [ ] Rename the images in S3 
* [ ] Drop this commit